test/CMakeLists.txt

```diff
@@ -15,6 +15,17 @@
 set (ranlux_24_48_test.pass_timeout_debug "900") # 15min
 set (ranlux_24_48_test.pass_timeout_release "720") # 12min
 
+if (WIN32)
+    list(APPEND ADDITIONAL_INCLUDE_DIRS ENV INCLUDE ENV CPATH)
+else()
+    list(APPEND ADDITIONAL_INCLUDE_DIRS ENV CPATH ENV C_INCLUDE_PATH ENV CPLUS_INCLUDE_PATH ENV INCLUDE_PATH)
+endif()
+
+find_path(HAVE_TBB NAMES tbb/tbb.h PATHS ADDITIONAL_INCLUDE_DIRS)
+if (_onedpl_gxx_version VERSION_GREATER_EQUAL 11 AND HAVE_TBB AND NOT ONEDPL_BACKEND MATCHES "^(tbb|dpcpp)$")
```

Comment:
`VERSION_GREATER_EQUAL`  is added in CMake version 3.7, so you probably should replace it with NOT VERSION_LESS

Reply:
I didn't check it, thanks for comment. I will make changes

Reply:
Now, the minimal version of CMake is 3.11, so this suggestion is not relevant anymore